docs/source/get-started/model-composition.rst

```diff
@@ -141,21 +39,21 @@ This example defines multiple models within the same Service, with separate APIs
         traffic={"timeout": 20},
     )
     class MultiModelService:
-        # Retrieve model references from the BentoML Model Store
-        model_a_ref = bentoml.models.get("model_a:latest")
-        model_b_ref = bentoml.models.get("model_b:latest")
+        # Retrieve model references from HF
+        model_a_path = HuggingFaceModel("model_a")
```

Comment:
Better to use huggingface model id form `<repo_name>/<model_ name>` or fake one to make users better know what `model_id` should be

Reply:
Updated. I changed them all to actual models.